external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DockerJDBCIntegrationSuite.scala

```diff
@@ -155,7 +155,7 @@ abstract class DockerJDBCIntegrationSuite extends SharedSparkSession with Eventu
       docker.startContainer(containerId)
       jdbcUrl = db.getJdbcUrl(dockerIp, externalPort)
       var conn: Connection = null
-      eventually(timeout(2.minutes), interval(1.second)) {
+      eventually(timeout(7.minutes), interval(1.second)) {
```

Comment:
@gaborgsomogyi . `7 minutes` is okay. But, it seems that you misunderstand my PR. You can use more higher values, but you should not increase the timeout of the other test suites. Please see my PR. I keep `2.minutes` for all the other test suite and only allow longer timeout for `OracleIntegrationSuite`.
- https://github.com/gaborgsomogyi/spark/pull/2/files#diff-bf03588f75a2d7b55214924fd611e528R99

Reply:
On the way...

Reply:
Hmmm, just read back the situation. [Here](https://github.com/apache/spark/pull/28893#discussion_r443809246) I've read 5 minutes what I've thought is a bit small so changed the code manually. Just seen your effort which contains 10 minutes :) Sorry...

Reply:
Thanks. Ya. During review, I didn't update the original comment. My bad. I should be more clear on the PR.